### PR TITLE
Add new tool skyline2isocor

### PR DESCRIPTION
**Pull request by llegregam**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-metabolomics/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by hechth** [comment]
@llegregam just out of curiosity, is the source code for the `skyline2isocor` package available somewhere?

**Comment by llegregam** [comment]
@hechth you can find the source code for skyline2isocor on github [here](https://github.com/MetaboHUB-MetaToul-FluxoMet/Skyline2IsoCor)

**Comment by llegregam** [comment]
@bgruening @lecorguille could I get a review please? :)

File: tools/skyline2isocor/skyline2isocor.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <requirement type='package' version='@TOOL_VERSION@'>skyline2isocor</requirement>
+    </requirements>
+    <command detect_errors='exit_code'><![CDATA[
+    skyline2isocor '$skyline_output' '$isocor_input' > '$stdout' 2> '$stderr'
+    ]]></command>
+    <inputs>
+        <param type='data' format='tabular, txt' name='skyline_output' label='Skyline output file' help='Skyline output to convert'/>
+    </inputs>
+    <outputs>
+        <data format='txt' name='stdout' label='Log'/>
```
Comment (bgruening) [inline tools/skyline2isocor/skyline2isocor.xml:15]: I would not expose stderr and stdout as separate files, Galaxy captures those differently.

But I guess this is a community decision.

**Review by bgruening — APPROVED** [review]

File: tools/skyline2isocor/skyline2isocor.xml (hunk 1 above)
Comment (lecorguille) [reply tools/skyline2isocor/skyline2isocor.xml:15]: Agree with this point.

Otherwise, It seems in a good shape for me.

Comment (llegregam) [reply tools/skyline2isocor/skyline2isocor.xml:15]: Ok, should I thus change the wrapper so that you can merge the PR?

**Pull request closed without merging**